<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FunctionPlotter</class>
 <widget class="QMainWindow" name="FunctionPlotter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FunctionPlotter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="equationLayout">
          <item>
           <widget class="QLabel" name="equationLabel">
            <property name="text">
             <string>Function</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="eqtnInput"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="minXLayout">
          <item>
           <widget class="QLabel" name="minXLabel">
            <property name="text">
             <string>Minimum X Value</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="minXInput"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="maxXLayout">
          <item>
           <widget class="QLabel" name="maxXLabel">
            <property name="text">
             <string>Maximum X Value</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="maxXInput"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="plotBtn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Plot</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QCustomPlot" name="display" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
